<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlaveFormClass</class>
 <widget class="QWidget" name="SlaveFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="lConnectMaster">
     <property name="text">
      <string>Связь с Мастером</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lColor">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>цвет</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lCountClient">
     <property name="text">
      <string>Кол-во клиентов</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lCount">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableWidget" name="table">
     <column>
      <property name="text">
       <string>Сессия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP::port</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Управление  башней</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="QPushButton" name="bLeft">
        <property name="text">
         <string>&lt;-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="bUp">
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QPushButton" name="bRight">
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bDown">
        <property name="text">
         <string>_</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>94</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
